toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -38,6 +37,19 @@ done
 mkdir -p $ISO_ROOT
 LABEL=CDROM
 
+function Retry {
+    local RETRY=0
+    local LIMIT=10
+    SLEEPSECONDS=15
+    while [ $RETRY -lt $LIMIT ]
+    do
+        "$@" && break
+        echo "$@" failed. Retrying... 
+        ((RETRY++))
+        sleep $SLEEPSECONDS
+    done
+}
```

Comment:
Let's use small caps for local variables. Otherwise they get confused with environmental or global ones.
```suggestion
    local retry=0
    local limit=10
    local sleep_seconds=15
    while [ $retry -lt $limit ]
    do
        "$@" && break
        echo "$@" failed. Retrying... 
        ((retry++))
        sleep $sleep_seconds
    done
}
```

Reply:
Fixed